### DISCUSSION TITLE
Error parsing config: 'mqtt'

### DISCUSSION THREAD

**Discussion by jkyselak** [blakeblackshear/frigate#745] [Ask A Question]
Sorry for another "stupid" question but after I added a mask to one of my cameras, I now get "Error parsing config: 'mqtt'" when I try to start the Frigate NVR. I've made no other changes and even after removing the new mask, I am still unable to start Frigate. MQTT is up and running and was not modified.
Any help is greatly appreciated

**Comment by paularmstrong** [comment]
Is this the same as one of these?

* #735 
* #736
* #703

**Reply by jkyselak** [reply]
Yes, looks that way, so I guess we need to wait for a solution?
